pkl-core/src/main/java/org/pkl/core/runtime/VmExceptionRenderer.java

```diff
@@ -73,7 +75,8 @@ private void renderBugException(VmBugException exception, StringBuilder builder)
   }
 
   private void renderException(VmException exception, StringBuilder builder) {
-    var header = "–– Pkl Error ––";
+    var out = ansi(builder);
+    out.render("@|red –– Pkl Error ––|@");
```

Comment:
Re: defining all colours in one place; this (and below) hard-coded redness definitely included in that.